crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -1337,8 +1337,10 @@ impl<'db> TypeInferenceBuilder<'db> {
         &mut self,
         _literal: &ast::ExprEllipsisLiteral,
     ) -> Type<'db> {
-        // TODO Ellipsis
-        Type::Unknown
+        // TODO: this does not currently resolve to `types.EllipsisType` as it
+        // should, and it needs support for sealed types (with a single member
+        // for singletons like this) as well (#12694).
```

Comment:
I would eliminate this TODO. This TODO will be resolved by other added features; the code right here shouldn't need to change at all, which means we also don't need a reminder TODO comment here.

```suggestion
```